rest-api/src/main/resources/paths/index.yml

```diff
@@ -197,8 +199,6 @@
     $ref: ./reports/v4_reports_identifier.yml
 
 # Surveys
-/v3/surveys:
-    $ref: ./surveys/v3_surveys.yml
 /v3/surveys/recent:
     $ref: ./surveys/v3_surveys.yml
```

Comment:
I think you want to delete the other one. GET /v3/surveys and GET /v3/surveys/recent both point to the same API on the server, but only POST /v3/surveys exists.